client-src/default/index.js

```diff
@@ -196,7 +196,7 @@ const socketUrl = url.format({
   auth: urlParts.auth,
   hostname,
   port: urlParts.port,
-  pathname: urlParts.path == null || urlParts.path === '/' ? '/sockjs-node' : urlParts.path
+  pathname: urlParts.path == null || urlParts.path === '/' ? '/sockjs-node' : (querystring.parse(urlParts.path).sockPath || urlParts.path)
```

Comment:
Why we need use `querystring.parse` here?

Reply:
That I don't know the answer to. It was left over from the initial pr and I kinda just assumed was how stuff was done. Will investigate and come back with an answer.

Reply:
@trescenzi thanks!

Reply:
So what's happening here is that `__resouceQuery` gets the `sockPath` option passed in. This is then parsing that querystring and getting the `sockPath` option.

Reply:
@trescenzi just add this comment to code :+1: